android-static-binding-generator/project/build.gradle

```diff
@@ -14,6 +14,15 @@ def cachedJarsFilePath = "$projectDir/cached.txt"
 def jsParserP = "$projectDir/parser/js_parser.js"
 def jsFilesParametersP = "$projectDir/jsFilesParameters.txt"
 
+def webpackWorkersExcludePath = "$projectDir/../../src/main/assets/app/__worker-chunks.json"
+def webpackWorkersExcludesList;
+
+def workersExcludeFile = file(webpackWorkersExcludePath);
+def filterWorkerFiles = false;
+if (workersExcludeFile.exists()) {
+	filterWorkerFiles = true;
+	webpackWorkersExcludesList = new JsonSlurper().parseText(workersExcludeFile.text)
+}
```

Comment:
Because if it's an app with webpack and the file does not exist, a warning can go a long way towards debugging the problem. In cases, users are dealing with webpack. I'm sure there's a way to add a check if the project is a webpack one and I believe it's important.